### PR TITLE
Multiplatform

### PR DESCRIPTION
**Pull request by kryptonite303**
This PR will add multiple platforms:
RHEL 6,7,8
Debian 9,10

Additionally it will be using GitHub Actions instead of CircleCI
Added new GitHub Actions Badges

### CODE REVIEW

File: molecule/default/prepare.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        name:
+          - sudo
+          - tar
+        state: present
+
+- gather_facts: false
+  become: true
+  name: Create
+  hosts: all
+  tasks:
```
Comment (jasonwalsh) [inline molecule/default/prepare.yml]: I don't think these tasks are needed for this role.

File: tasks/main.yml [hunk 2]
```diff
@@ -5,3 +5,16 @@
     remote_src: true
     src: "{{ curator_url }}"
   become: true
+
+- name: Check if /usr/bin exists
+  stat:
+    path: /usr/bin
+  register: usr_bin
+
+- name: Link curator binary to common places in PATH
```
Comment (jasonwalsh) [inline tasks/main.yml]: Which platforms don't include `/usr/local/bin` in the `PATH`?

Comment (jasonwalsh) [reply tasks/main.yml]: It seems strange to me that we check for the existence of `/usr/bin` before running the symlink task. I think a simple `ignore_errors: true` directive would suffice in the event that the `/usr/bin` directory does not exist.

Comment (kryptonite303) [reply tasks/main.yml]: The environment greenbay tests run in for some reason don't have `/usr/local/bin` in the path by default.

File: molecule/default/prepare.yml (hunk 1 above)
Comment (kryptonite303) [reply molecule/default/prepare.yml]: I removed the efs tasks that aren't used.

File: tasks/main.yml [hunk 3]
```diff
@@ -6,15 +6,10 @@
...
-  register: usr_bin
-
 - name: Link curator binary to common places in PATH
   become: true
   file:
     state: link
     path: /usr/bin/curator
     src: /usr/local/bin/curator
-  when: usr_bin.stat.exists
+  ignore_errors: true
```
Comment (jasonwalsh) [inline tasks/main.yml]: lookin GOOD

**Pull request merged**